### ISSUE TITLE
[mysql] Dynamic attributes should not be assigned to the ADOfieldObject object of mysql_fetch_field() in ADORecordset::Fields()

### ISSUE THREAD

**Issue by mnewnham** [ADOdb/ADOdb#906]
```
Deprecated: Creation of dynamic property ADOFieldObject::$orgname is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$table is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$orgtable is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$def is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$db is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$catalog is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$length is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$charsetnr is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$flags is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$decimals is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$auto_increment is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$binary is deprecated in drivers/adodb-mysqli.inc.php on line 1487

Deprecated: Creation of dynamic property ADOFieldObject::$unsigned is deprecated in drivers/adodb-mysqli.inc.php on line 1487
```

**Comment by dregad** [comment]
Due to the inherently dynamic nature of ADOFieldObject class, I currently do not see any alternative but to add the `#[\AllowDynamicProperties]` attribute. 

I'll fix this one like that for now, if you have a better idea please let me know.

**Comment by dregad** [comment]
Fixed in d085fc8de14065d11b2183b2a48eeafe3f1d3fe4
